lyse/__init__.py

```diff
@@ -43,29 +38,18 @@
 
 # Import this way so LYSE_DIR is exposed when someone does import lyse or from lyse import *
 from lyse.utils import LYSE_DIR
-
-# If running stand-alone, and not from within lyse, the below two variables
-# will be as follows. Otherwise lyse will override them with spinning_top =
-# True and path <name of hdf5 file being analysed>:
-spinning_top = False
-# data to be sent back to the lyse GUI if running within lyse
-_updated_data = {}
-# dictionary of plot id's to classes to use for Plot object
-_plot_classes = {}
-# A fake Plot object to subclass if we are not running in the GUI
-Plot=object
-# An empty dictionary of plots (overwritten by the analysis worker if running within lyse)
-plots = {}
-# A threading.Event to delay the 
-delay_event = threading.Event()
-# a flag to determine whether we should wait for the delay event
-_delay_flag = False
-
-if len(sys.argv) > 1:
-    path = sys.argv[1]
-else:
-    path = None
```

Comment:
Looking back over this, I have a concern that we are removing a workflow (ie running `python analysis.py h5_path.h5` to run single shot analyses without needing to run lyse. Personally I don't think it is much of a loss, but I also don't want to break too much if people actually use that workflow.

Supporting it will be slightly annoying, but certainly possible. @ispielma Do you have any input?

Reply:
I don't think we should support this.  My argument is that the whole way lyse works to modify the namespace is a really bad idea (TM) anyway.  One of the back-end things I am working on is adding a second lyse workflow that removes all the magic that is currently under the hood (with the plan to do the same to the format of labscript scripts.

So I'd rather spend time on this rather than yet more magic under the hood to support a potential use case.

Reply:
I am inclined to agree. The inter-process communication here isn't my favorite. It is probably sporting to add a helpful hint in case someone actually does this so they know who to blame for borking their workflow.

Though the more I think about the changes I've made, I'd like to modify how `path` is bodged. One of my personal goals is to remove all star import from labscript, including in examples and docs. `from lyse import *` is a common paradigm, and the code I wrote currently would have things like `run = lyse.Run(path)`, which mixes the magical syntax. I think I'd just as soon keep the underlying machinery but re-expose `path` as `lyse.path` so people at least are clear on where the magic is coming from.

I'll make another PR against your branch.

Reply:
@ispielma I've made a few final tweaks that ensure existing star imports do not break. While I really want to nuke star imports, breaking them without warning doesn't feel right. 

Anyway, we've been testing these changes for a bit without issues, so I'm happy to merge. If you can merge my PR to yours, that would be great. Otherwise, I'll go ahead and merge as-is and move my patches over directly.